driver/configuration/provider_viper.go

```diff
@@ -228,6 +243,14 @@ func (v *ViperProvider) adminPort() int {
 	return viperx.GetInt(v.l, ViperKeyAdminListenOnPort, 4445, "ADMIN_PORT")
 }
 
+func (v *ViperProvider) AdminSocketPermission() *UnixPermission {
```

Comment:
Could you add a test for this please? It doesn't have to be too complicated, you can look for examples over here: https://github.com/ory/hydra/blob/master/driver/configuration/provider_viper_test.go#L209 :)